systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -105,9 +132,7 @@ public abstract class AbstractST implements TestSeparator {
     }
 
     // Test-Frame integration stuff, remove everything else when not needed
-    protected final KubeResourceManager kubeResourceManager = KubeResourceManager.get();
-
-    protected final ResourceManager resourceManager = ResourceManager.getInstance();
+    protected final KubeResourceManager resourceManager = KubeResourceManager.get();
```

Comment:
I think even if it shortens the code a bit for all the classes extending off of AbstractST and it's fine in this case, you yourself advised me against the usage, when there is the static `KubeResourceManager.get();` for consistency. So this is maybe more of a question, are you okay with that, if so i have no issue with having it here.

Reply:
No strong opinion, I kept it there because I didn't want to change it everywhere else. But I guess it's connected to https://github.com/strimzi/strimzi-kafka-operator/pull/11353#discussion_r2055580093 comment and I can simply remove this.